### DISCUSSION TITLE
Load local file

### DISCUSSION THREAD

**Discussion by Chance-Obondo** [fastapi/fastapi#6136] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# THE ROOT API
# the data is a list of dictionaries
@app.get("/")
async def root():
    data = open('valid.json')
    json_data = json.load(data)

    return json_data[0]
```

### Description

I am trying to load a local json file on my computer using my fast API main.py file. But I keep on getting a file not found error.
What could be the issue and how can I solve this problem considering I need to use the json content in the file within the app itself and not return it to an end user

### Operating System

Windows

### Operating System Details

Chrome OS

### FastAPI Version

0.85.0

### Python Version

3.8.12

**Comment by iudeen** [comment]
Does it work without FastAPI? As in just as a normal function?

**Comment by JarroVGIT** [comment, marked as answer]
Typically, this means you are not referencing the right subfolder.  You can try the following:
```
import os

@app.get("/")
async def root():
    return os.getcwd()
```
This will give the current working directory, and probably that is a few folders above your file.

**Comment by Chance-Obondo** [comment]
@JarroVGIT It is working now correct;y. Thanks very much

**Comment by tiangolo** [comment]
Thanks for the help here @JarroVGIT and @iudeen  ! 👏  🙇 

Thanks for reporting back and closing the issue @Chance-Obondo  👍
